.github/workflows/fast-forward-candidate-branch.yml

```diff
@@ -0,0 +1,16 @@
+name: Fast-forward candidate branch to HEAD of default branch
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'The release number'
+        required: true
+        default: '2.11'
+
+jobs:
+  fast-forward:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: git push origin HEAD:3scale-${{ github.event.inputs.release }}-candidate
```

Comment:
HEAD refers here to the branch against which you run the workflow.

So maybe make it configurable and pointing to the HEAD of main branch by default